photon-client/src/components/app/photon-3d-visualizer.vue

```diff
@@ -0,0 +1,211 @@
+<script setup lang="ts">
+import type { PhotonTarget } from "@/types/PhotonTrackingTypes";
+import { onBeforeUnmount, onMounted, watchEffect } from "vue";
+import {
+  ArrowHelper,
+  BoxGeometry,
+  ConeGeometry,
+  Mesh,
+  MeshNormalMaterial,
+  PerspectiveCamera,
+  Quaternion,
+  Scene,
+  Vector3,
+  Color,
+  WebGLRenderer
+} from "three";
+import { TrackballControls } from "three/examples/jsm/controls/TrackballControls";
+import { type Object3D } from "three";
+
+const props = defineProps<{
+  targets: PhotonTarget[],
+  horizontalFOV: number // TODO this is unused, can it be removed?
```

Comment:
it used to draw a cone in the top-down 3d view. Now that we use a fully 3d engine we can yeet it, or we can use the diagonal FOV to set the size of the cone coming out of the camera model